containers/roles/foreman/files/create_qpid_queue.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-qpid-config -b tcp://${QPID_SERVICE}:${QPID_PORT} queues katello_event_queue
+qpid-config -b ${QPID_PORT} queues katello_event_queue
 
 if [ ?! != 0 ]; then
-  qpid-config -b tcp://${QPID_SERVICE}:${QPID_PORT} add queue katello_event_queue --durable
+  qpid-config -b ${QPID_PORT} add queue katello_event_queue --durable
```

Comment:
Is this supposed to be the built-in variable? I think it's `QPID_SERVICE_PORT` / `QPID_SERVICE_HOST`

Reply:
Just deployed and this is what I have in the foreman container for QPID:

```bash
sh-4.2# env | sort | grep -i qpid
QPID_PORT=tcp://172.30.219.102:5672
QPID_PORT_5672_TCP=tcp://172.30.219.102:5672
QPID_PORT_5672_TCP_ADDR=172.30.219.102
QPID_PORT_5672_TCP_PORT=5672
QPID_PORT_5672_TCP_PROTO=tcp
QPID_SERVICE_HOST=172.30.219.102
QPID_SERVICE_PORT=5672
QPID_SERVICE_PORT_PORT_5672_TCP=5672
```

Reply:
I was also surprised that QPID_PORT had IP and PORT but it seems to be a consistent pattern.